careerfairsystems/arkad-app
# Qr scanner

## Change description

Adds a QR-reader to when you are logged on in profile. Should work on android. Needs fine-tuning for iOS

## How to verify

When logged on in profile, a camera button should be visible. Clicking should request permission and after permission is granted, navigate to CameraScreen.

Probably doesn't need to be verified, but scanning a QR-code will navigate you to the FAQ screen (as a placeholder for now, later we will change it to add a student)

## Issues fixed
#199
